test/escape-test.js

```diff
@@ -49,6 +49,9 @@ vows.describe('escape').addBatch({
     'unescapes \'': function () {
       assert.strictEqual(unescapeXML('&apos;'), '\'')
     },
+    'leaves invalid entities alone': function () {
+      assert.strictEqual(unescapeXML('&foobar;'), '&foobar;')
+    },
```

Comment:
should we throw instead?

Reply:
That was my first inclination, and my first implementation, but I didn't want to break backwards compatibility, so I decided to write a test that verified that the behaviour didn't change.  If anything, that might be better to put in a different PR.

Reply:
sounds sane :) good candidate for release 3.0